### ISSUE TITLE
FMD randomdly can't see mangadex titles with 0.9.157.0 version (0.156.o patched worked fine)

### ISSUE THREAD

**Issue by Black-Dalek** [riderkick/FMD#1582]
![imagen](https://user-images.githubusercontent.com/36430844/50535002-4400ea80-0b0a-11e9-83a5-1de07c409e83.png)
Now i can see & download mangafox files, but not all titles on mangadex, only random titles.

**Comment by kmvi** [comment]
is the [last fix](3a63747a5514d972fe9b3582b3c5024158cd2c62) downloaded? if this does not help, call "view manga info" from the context menu on such row and post the title url here

**Comment by Black-Dalek** [comment]
@kmvi, yes, it was downloaded before upgrade to .157.0, but had this text on "last message" cell: "mangadex, fix, use api", i read your response, so  i deleted the lua file (i checked date file and it was  28/12/18) and let FMD download it again, this time the last message say "mangadex, fix", and it works fine again. thank you!.
